src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -755,6 +761,12 @@ StDebugger >> initializePresenters [
 	self layout: self defaultLayout
 ]
 
+{ #category : 'initialization' }
+StDebugger >> initializeSettings: aWindowPresenter [
+
+	self debuggerSettings initializeSettings: aWindowPresenter
+]
```

Comment:
the call to the settings should be inlined in the method needing it.
here there is no reason that the parameter would be a window presenter

Reply:
hence I would remove that method entirely

Reply:
Done